values.yaml

```diff
@@ -77,3 +77,69 @@ storage:
   elasticsearch:
     port: 9200
     host: 127.0.0.1
+
+exporter:
+  enabled: false
+  image:
+    repository: registry.ng.bluemix.net/secadvisor_dev/skydive-flow-exporter
```

Comment:
Eventually this should be something like: `skydive/skydive-flow-exporter` and tag: `1.0` but this depends on the CI system creating and publish these docker images to the main docker hub.

Reply:
Also I don't think the name `repository` is good. I think it's `image` and can be simply one field (no need for `repository` and `tag` fields):

    image: skydive/skydive-flow-exporter:1.0

Reply:
image:
  repository:
  tag: 
is made similar to the main skydive image definitions..